### ISSUE TITLE
Pivot elements are not clearly seen while TABing through the Pivot headers in High-Contrast mode.

### ISSUE THREAD

**Issue by dhiv-nitt** [OfficeDev/office-ui-fabric-react#11966]
### Environment Information

Package version: 6.197.0
OS version: Window 10
Browser: Microsoft Edge 44.18362.1.0
Screen reader: Narrator

### Describe the issue:

When we move over the Pivot headers through keyboard navigation, we are unable to see the Pivot header highlighted in High-contrast mode.

#### Actual behavior:

Pivot header not highlighted when we TAB over it when we TAB over it in High-contrast mode.

![MAS431-Tab-elements](https://user-images.githubusercontent.com/47556452/74636560-43079e00-518e-11ea-9e19-80b754f82932.gif)

#### Expected behavior:

Pivot header should have a significant highlighted border when we TAB over it in High-contrast mode.

**Comment by khmakoto** [comment]
Hi @dhiv-nitt, are you sure this does not have to do with some styling that you're putting in? I just checked version 6.197.0 in our website and this is what I see:

![Pivot](https://user-images.githubusercontent.com/7798177/74772808-8d982180-5245-11ea-8b61-49a65a8364ab.gif)

**Comment by dhiv-nitt** [comment]
@khmakoto Let me check with my styling details and get back to you on this within a days's time.

**Comment by dhiv-nitt** [comment]
Hi @khmakoto, you were right ... made a few styling changes and it worked. thank you ! Closing the issue !
